docs/daemon/configuration/genesis.mdx

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 2
+sidebar_label: "Genesis"
+id: daemon-genesis
+title: Genesis
+description: Genesis file configuration
+slug: /daemon/genesis
+---
+Genesis file in `rootdir/abci/config/` defines the initial kwild state upon the initialization of the kwild network
```

Comment:
Nit pick on the grammar here "The genesis file...", as well as punctuation.  We should also go into a bit more depth about the genesis; that it includes the initial set of validators, what to do if you are running your own network, etc.

Also, since the path is configurable, we probably do not need to mention the path here

Reply:
I have a different section for going in-depth regarding the setup, so left it out from here. Probably I will reference the other page here